docs/source/optimizations/ibm/ibmq_compile_css.ipynb

```diff
@@ -22,21 +22,27 @@
    ]
```

Comment:
<p>Typo: should be passed as an argument to css.Service()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Fixed.